src/main/java/model/figure/Circle.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (Math.PI * (radius * radius));
```

Comment:
Consider using a constant for the value of PI to avoid magic numbers. While 'Math.PI' is a well-known constant, defining it as a static final variable within the class can improve readability and maintainability.